ui/objects/semantic-ui/js/objects/editform/components/BasicInfo.jsx

```diff
@@ -73,79 +73,57 @@ export const BasicInfo = ({ activeIndex, handleActive, record }) => {
             name="metadata.restorationObject.keywords"
             disabled={false}
           />
-        </div>
-        <div className="vert-div predmety__form__div">
+        </Grid.Column>
+        <Grid.Column>
           <TextAreaField
             name="metadata.restorationObject.description"
             aria-label="Popis"
             fieldPath="metadata.restorationObject.description"
             label={
               <FieldLabel
                 htmlFor="metadata.restorationObject.description"
-                className="predmety__form__div__label"
                 label="Popis"
               ></FieldLabel>
             }
           />
-        </div>
-
-        <div className="vert-div predmety__form__div">
-          <LocalVocabularySelectField
+        </Grid.Column>
+        <Grid.Column>
+          <VocabularyTreeSelectField
             optionsListName="ItemTypes"
             fieldPath="metadata.restorationObject.itemTypes"
             multiple={true}
             placeholder="Vyberte typ předmětu"
+            category={category}
             label={
               <FieldLabel
                 htmlFor="metadata.restorationObject.itemTypes"
                 label="Typ předmětu"
-                className="predmety__form__div__label"
               />
             }
           />
-        </div>
-        <div>
-          <div className="horiz-div predmety__form__div">
-            <div className="vert-div predmety__form__div-medium">
-              <TextField
-                name="metadata.restorationObject.creationPeriod.since"
-                aria-label="Počateční rok"
-                fieldPath="metadata.restorationObject.creationPeriod.since"
-                placeholder="Počateční rok"
-                label={
-                  <FieldLabel
-                    htmlFor="metadata.restorationObject.creationPeriod.since"
-                    className="predmety__form__div__label"
-                    label="Počateční rok"
-                  ></FieldLabel>
-                }
-              />
-            </div>
-            <div className="vert-div predmety__form__div-medium">
-              <TextField
-                name="metadata.restorationObject.creationPeriod.until"
-                aria-label="Končící rok"
-                fieldPath="metadata.restorationObject.creationPeriod.until"
-                placeholder="Končící rok"
-                label={
-                  <FieldLabel
-                    htmlFor="metadata.restorationObject.creationPeriod.until"
-                    className="predmety__form__div__label"
-                    label="Končící rok"
-                  ></FieldLabel>
-                }
-              />
-            </div>
-          </div>
-        </div>
-        <div className="vert-div predmety__form__div-dimensions">
+        </Grid.Column>
+
+        {/* <Grid.Column>
+          <EDTFDateRangePickerField
```

Comment:
EDTFDate picker is now released under https://github.com/oarepo/oarepo-ui/releases/tag/5.1.3